src/components/pages/SoraCard/steps/Payment.vue

```diff
@@ -108,7 +123,7 @@ export default class Payment extends Mixins(TranslationMixin, mixins.LoadingMixi
 }
 </script>
 
-<style lang="scss" scope>
+<style lang="scss">
```

Comment:
Why did you remove it? I don't see any reason

Reply:
it's mistake created long ago, proper attributed is 'scoped'. Should be non-scoped